content/en/docs/ibm/install-kubeflow.md

```diff
@@ -36,37 +36,46 @@ environment variables.
     ```
     export CLUSTER_NAME=kubeflow
     export CLUSTER_ZONE=dal13
+    export WORKER_PROVIDER=classic
     ```
 
     - `CLUSTER_NAME` must be lowercase and unique among any other Kubernetes
       clusters in the specified CLUSTER_ZONE.
-    - `CLUSTER_ZONE` identifies the location where CLUSTER_NAME will be created. Run `ibmcloud ks locations` to list supported IBM Cloud Kubernetes Service locations. For example, choose `dal13` to create CLUSTER_NAME in the Dallas (US) data center. 
+    - `CLUSTER_ZONE` identifies the location where CLUSTER_NAME will be created. Run `ibmcloud ks locations` to list supported IBM Cloud Kubernetes Service locations. For example, choose `dal13` to create CLUSTER_NAME in the Dallas (US) data center.
```

Comment:
```suggestion
    - `CLUSTER_ZONE` identifies the regions or locations where CLUSTER_NAME will be created. Run `ibmcloud ks locations` to list supported IBM Cloud Kubernetes Service locations. For example, choose `dal13` to create CLUSTER_NAME in the Dallas (US) data center.
```

Reply:
addressed in https://github.com/kubeflow/website/pull/2028/commits/9c471a4aecf999fdeb136ae19e68240ba9b290cb